utils/docker_nlt.sh

```diff
@@ -6,8 +6,32 @@
 
 set -e
 
+set -x
+
 . utils/sl/setup_local.sh
 
-./utils/setup_daos_admin.sh
+sudo --preserve-env=SL_PREFIX ./utils/setup_daos_admin.sh
+
+TMP_DIR=`mktemp -d`
+
+cp utils/node_local_test.py utils/nlt_server.yaml utils/nlt_agent.yaml .build_vars.json src/tests/ftest/cart/util/cart_logparse.py src/tests/ftest/cart/util/cart_logtest.py $TMP_DIR
```

Comment:
(style)  line over 100 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]